test/AccountMgmt/AccountMgmtTest.php

```diff
@@ -82,6 +82,45 @@ public function testSTagListInvalidCustomerKey() {
    } // End function testShowNotFound()
 
    /**
+    * Tests create, edit, and view customer key
+    */
+   public function testCreateEditViewKey() {
```

Comment:
Yes, I was going to create separate "testCreateEditKey()" and "testCreateViewKey()" functions but since you can only validate and edit with a view it made sense to combine them.